axelrod/strategies/selfsteem.py

```diff
@@ -0,0 +1,50 @@
+from axelrod.actions import Action, Actions
+from axelrod.player import Player
+from axelrod.random_ import random_choice
+
+from math import pi, sin
+
+C, D = Actions.C, Actions.D
+
+
+class SelfSteem(Player):
+    """
+    This strategy is based on the feeling with the same name.
```

Comment:
It means self esteem, that's how its named in the paper. I decided not to change it.